utils/dist_build.sh

```diff
@@ -1,25 +1,30 @@
 #!/bin/bash
+: ${ORGDIR:="/src/github.com/TykTechnologies"}
+: ${SOURCEBINPATH:="${ORGDIR}/tyk"}
+: ${SIGNKEY:="729EA673"}
+: ${BUILDPKGS:="1"}
+BUILDTOOLSDIR=$SOURCEBINPATH/build_tools
+
 echo Set version number
 export VERSION=$(perl -n -e'/v(\d+).(\d+).(\d+)/'' && print "$1\.$2\.$3"' version.go)
 
 echo Generating key
-[[ $(gpg --list-keys | grep -w 729EA673) ]] && echo "Key exists" || gpg --import build_key.key
+
+[[ $(gpg --list-keys | grep -w $SIGNKEY) ]] && echo "Key exists" || gpg --import $BUILDTOOLSDIR/build_key.key
```

Comment:
I know this was like this before, but could this not be `foo && bar || etc` instead of `[[ $(foo) ]] && bar || etc`?